services/tier-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/dal/ddb/DynamoTierDataStore.java

```diff
@@ -74,7 +90,7 @@ public void deleteTier(String id) {
     }
 
     @Override
-    public void updateTier(Tier tier) throws TierNotFoundException {
+    public Tier updateTier(Tier tier) throws TierNotFoundException {
```

Comment:
We don't use checked exceptions anywhere else in SaaS Boost. I'd prefer to make TierNotFoundException a runtime exception. The service code can still catch it and manage it as it wants, but let's make this a runtime exception.